test/examples/.eslintrc.js

```diff
@@ -0,0 +1,54 @@
+module.exports = {
+    'extends': [
+        'eslint-config-airbnb-base',
+        'eslint-config-airbnb-base/rules/strict',
+    ],
+    parserOptions: {
+        ecmaVersion: 2017,
+        sourceType: 'script',
+        ecmaFeatures: {
+            impliedStrict: true
+        }
+    },
+    env: {
```

Comment:
Add Mocha? https://eslint.org/docs/user-guide/configuring#specifying-environments (IIUC, that'll save you from manually declaring its globals)

Reply:
Ah good idea, thanks